sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -25,10 +25,49 @@ WITH overactive AS (
   HAVING
     COUNT(*) > 200000
 ),
-client_experiments AS (
+client_map_sums AS (
   SELECT
     client_id,
     udf.map_mode_last(ARRAY_CONCAT_AGG(experiments)) AS experiments,
+    udf.map_sum(
```

Comment:
I think `mozfun.map.mode_last` and `mozfun.map.sum` would be preferred for lower complexity.